### PR TITLE
fixing: refunded order smart contract not trigger

### PR DESCRIPTION
**Pull request by JackyRahman**
### JIRA LINK
- https://blocksphere2020.atlassian.net/browse/DBIO-2491
- https://blocksphere2020.atlassian.net/browse/DBIO-2571
- https://blocksphere2020.atlassian.net/browse/DBIO-2689

### Changelog
- [fix: refund order contract](https://github.com/debionetwork/debio-backend/commit/a373ca780b45ac68176988f949f60923496d173b)

### CODE REVIEW

File: src/endpoints/email/email.controller.ts [hunk 1]
```diff
@@ -68,6 +71,21 @@ export class EmailEndpointController {
...
+      description: `You've successfully submitted your account verification.`,
+      read: false,
+      created_at: await new Date(),
+      updated_at: await new Date(),
+      deleted_at: null,
+      from: 'Debio Network',
+      to: lab_id,
+    };
+
+    await this.notificationService.insert(notificationRegistration);
```
Comment (agustinustheo) [inline src/endpoints/email/email.controller.ts:87]: Please remove this.

Comment (agustinustheo) [inline src/endpoints/email/email.controller.ts:87]: Please remove this.

**Review by agustinustheo — CHANGES_REQUESTED** [review]

File: src/endpoints/email/email.controller.ts (hunk 1 above)
Comment (JackyRahman) [reply src/endpoints/email/email.controller.ts]: removed

Comment (JackyRahman) [reply src/endpoints/email/email.controller.ts]: removed

**Review by agustinustheo — APPROVED** [review]

**Pull request merged**